code/modules/mining/mine_items.dm

```diff
@@ -29,6 +29,7 @@
 	sharp = 1
 
 	var/excavation_amount = 200
+	var/destroy_artefacts = FALSE // some mining tools will destroy artefacts completely while avoiding side-effects.
```

Comment:
As per the March 3rd Staff Meeting, we like everything in this PR other than this particular feature. If you'd rather leave it in, at least disable it on the assorted drills.

Reply:
It's disabled by default on the standard drill (not sure at what point regular pickaxes became drills) - which things, if anything, would you prefer it be enabled on?

Reply:
Didn't mean to resolve that. We'd rather it not be active on anything, for the time being.

Reply:
Alrighty. Probably gonna enable it on some things downstream because it was a major headache for expeditions given that we have a lot of underground exploration maps.

Reply:
Makes perfectly good sense, in your situation, which is why it makes sense to me to leave the code proper in.